.travis.yml

```diff
@@ -57,9 +57,9 @@ jobs:
         - rm -rf $HOME/.ivy2
         - cd $TRAVIS_BUILD_DIR && sbt clean stage
         # only uncomment if docker image is needed for this branch. remove before merging into develop
```

Comment:
The comments says "remove before merging into develop". :)

Reply:
haha, Ivan has ignored this comment 😄 I will fix it